libs/api/app/data-access/src/lib/entity/app-transaction.entity.ts

```diff
@@ -21,7 +21,9 @@ export class AppTransaction {
   @ApiProperty()
   @Field({ nullable: true })
   destination?: string
-  @ApiProperty()
+  @ApiProperty({
+    type: [AppTransactionError],
```

Comment:
@RamirezAlex this was needed to make the `errors` property typed properly in the `AppTransaction` model in Swagger.